### ISSUE TITLE
Feature request: extend valid-title to include tags

### ISSUE THREAD

**Issue by aaron-goff** [playwright-community/eslint-plugin-playwright#342]
Use case: We often use an `@new` tag when writing new tests to help us focus down on just running those new tests, before removing the tag prior to creating our PRs.

It makes most sense to me to extend the `valid-title` rule to include support for this, possibly with a `disallowedTags` option? I can see it also being it's own rule if that makes more sense.

```js
...
        'playwright/valid-title': [
          'error',
          {
            disallowedTags: ['@new']
          }
        ],
...
```
